sdk/networkanalytics/arm-networkanalytics/package.json

```diff
@@ -119,126 +119,154 @@
     "./package.json": "./package.json",
     ".": {
       "browser": {
+        "source": "./src/index.ts",
```

Comment:
do you know why we have this change?

Reply:
@jeremymeng this added line `source` is happened when we run build in sdk repo. is this expected?

Reply:
Yes, I think it's because of some changes in newer version of `tshy`.  From @mpodwysocki 

> It's a new feature inside tshy, see the [exports section](https://github.com/isaacs/tshy?tab=readme-ov-file#exports)

https://github.com/Azure/azure-sdk-for-js/pull/30011#discussion_r1636681439